package.json

```diff
@@ -113,7 +113,7 @@
     "preact": "^7.1.0",
     "preact-compat": "^3.9.0",
     "raven-js": "^3.10.0",
-    "react-bosonic": "^1.0.0-beta5",
+    "react-bosonic": "beta",
```

Comment:
can you rebase your PR and remove this change ;) https://github.com/cozy/cozy-files-v3/pull/141